packages/lib/src/user/utils.ts

```diff
@@ -233,7 +233,13 @@ const settingIsOverridden = (
  * @returns the campus name and campus code that the user is associated with
  */
 const usersCampus = (user: User): { campusName: string | undefined; campusCode: string } => {
-  const campusCode = user.audienceOverride?.campusCode || user.classification?.attributes?.campusCode || DEFAULT_CAMPUS;
+  let campusCode = user.audienceOverride?.campusCode || user.classification?.attributes?.campusCode || DEFAULT_CAMPUS;
+
+  // Check to see if user is employee who used to be a student, return either audience override or default campus
+  if (hasPrimaryAffiliation(user, ['employee']) && isStudent(user)) {
```

Comment:
I like using a `let` here, but this scenario is a little tricky.. this particular user has `affiliations: ['employee', 'member']` but not `student`, but they _do have student classification data_ which is what causes this edge-case to happen.. so I think what you'd want in this if-statement is;

`if(isEmployee(user) && user.classification?)` 

Might even be worthwhile adding a new method for this strange scenario.. could be that this if-statement becomes it's own method `employeeFormerStudent`, so that we might use it in other logic as well if necessary. Up to you if you wanna take it this far.